example_audit.rules

```diff
@@ -157,19 +183,32 @@ w /etc/fluent/ -p wa -k syslog
 -w /usr/local/libexec/ -p wa -k binaries
 
 # Important system files
--w /etc/sysctl.conf -p wa -k binaries
--w /etc/modprobe.conf
--w /etc/modprobe.conf.local
--w /etc/modprobe.d/ -p wa -k binaries
--w /etc/modules-load.d/ -p wa -k binaries
+-w /etc/sysctl.conf -p wa -k kernel
+-w /etc/sysctl.d -p wa -k kernel
+-w /etc/modprobe.conf -p wa -k kernel
+-w /etc/modprobe.conf.local -p wa -k kernel
+-w /etc/modprobe.d/ -p wa -k kernel
+-w /etc/modules-load.d/ -p wa -k kernel
 
 # Kernel modules
--w /lib/modules/ -p wa -k binaries
+-w /lib/modules/ -p wa -k kernel
+
+# Critical devices
+-w /dev/mem -p wa -k kernel
```

Comment:
mknode /dev/other-name-here ;-)
is this really that useful?

Reply:
Indeed - removed. kmem is disabled by default on any modern distribution, mem gives read-only access and if someone has privileges to access ports/iopl/ioperm/msr they can just backdoor the kernel.